pom.xml

```diff
@@ -1585,6 +1585,14 @@
           <groupId>net.alchim31.maven</groupId>
           <artifactId>scala-maven-plugin</artifactId>
           <version>${plugin.scala.alchim31.version}</version>
+          <configuration>
+            <recompileMode>all</recompileMode>
```

Comment:
is it necessary? incremental mode should be faster

Reply:
Agreed with the performance. But it causes failure as it shows an error when zinc is not downloaded for the first time. So I checked it. Moreover, we are developing scala code but it's not a heavy job so I think it should be fine. Otherwise, we can introduce some CI profile and enable it on it but I don't know if it's really necessary